src/Language/Instance.hs

```diff
@@ -870,11 +879,4 @@ prettyEntityTable alg@(Algebra sch en' _ _ _ _ _ _ _) es =
     -- prettyAtt :: x -> (att, w) -> String
     prettyAtt x (att,_) = prettyTerm $ aAtt alg att x
 
-    prettyTerm = show
-
-instance TyMap Show '[var, ty, sym, en, fk, att, gen, sk]
-  => Show (Presentation var ty sym en fk att gen sk) where
-  show (Presentation ens' _ eqs') =
-    indentLines $
-      "generators\n\t" ++ showCtx' ens' ++ "\n" ++
-      "equations\n\t" ++ intercalate "\n\t" (Set.map show eqs') ++ "\n"
+    prettyTerm = show
```

Comment:
newline at the end of the file

Reply:
Addressed here: https://github.com/statebox/cql/pull/150/files#diff-8687d19161f31f31db2c6853a2f8781aR890.